test/test.js

```diff
@@ -29,7 +31,31 @@ tape('handle undefined style_name', function(t) {
     });
 });
 
-tape('font machine 256-511', function(t) {
+test('handle font not found', function(t) {
+    fontmachine.makeGlyphs({font: 'NOT-FOUND', filetype: '.ttc'}, function(err, font) {
```

Comment:
So... this is just passing a string directly through to `fontnik.load`, when the API requires passing a `Buffer`. Adding some argument validation (`opts.font instanceof Buffer && opts.callback instanceof Function`) at the beginning of [`makeGlyphs`](https://github.com/mapbox/fontmachine/blob/9aabe5d98f3820296b2712c92777b59add3c2477/index.js#L19) seems like a better approach than `try/catch`  around `fontnik.load` or trying to change this upstream to pass an error through to the callback? If an attempt to read a font from disk fails in a downstream module, shouldn't that error be caught before trying to pass along something invalid to `fontmachine`?